silkworm/silkrpc/json/types_test.cpp

```diff
@@ -271,6 +271,7 @@ TEST_CASE("serialize block with baseFeePerGas", "[silkrpc][to_json]") {
     silkrpc::Block rpc_block {
         std::vector<silkworm::Transaction> {},
         std::vector<silkworm::BlockHeader> {},
+        std::nullopt,
```

Comment:
Given that you're changing this declaration, it would be a good idea to use structure binding syntax to improve readability